internal/datacoord/server.go

```diff
@@ -572,6 +572,7 @@ func (s *Server) handleTimetickMessage(ctx context.Context, ttMsg *msgstream.Dat
 	ch := ttMsg.GetChannelName()
 	ts := ttMsg.GetTimestamp()
 	physical, _ := tsoutil.ParseTS(ts)
+	log.Info("handleTimetickMessage triggered", zap.String("channel_name", ch), zap.Uint64("timestamp", ts), zap.String("physical_time", physical.String()))
```

Comment:
`Info` level maybe too high for this log. Any this log could be very frequent, do we have any alternatives place to print need information?

Reply:
Let me downgrade it to debug level